File: Java/Exercicio13.java
//Algoritmo que lê o código e horas trabalhadas de um operário e calcula o salário,
//sabendo que ele ganha R$10 por hora. Quando o número de horas exceder 50, calcula
//o excesso de pagamento e armazena. Caso contrário, ele zera.
//A hora excedente de trabalho vale R$20. No final mostra o salário total e excedente. 

package com.exercicios;

import java.util.Scanner;

public class Exercicio13 {
    public static void main(String[] args) {
        //Variáveis
        int codigo;
        float horas_trab, salario, excesso = 0, valor_hora = (float) 10.00f;
        Scanner entrada = new Scanner(System.in);

        //Entrada
        System.out.println("Informe o código: ");
        codigo = entrada.nextInt();
        System.out.println("Informe a quantidade de horas trabalhadas: ");
        horas_trab = entrada.nextFloat();

        //Processamento
        if (horas_trab > 50) {
            excesso = (horas_trab - 50) * 20.00f;
        }

        salario = 50 * valor_hora + excesso;

        //Saída
        System.out.printf("Código: %d \n", codigo);
        System.out.printf("O salário total é: R$ %.2f \n", salario);
        System.out.printf("O salário excedente é: R$ %.2f \n", excesso);

        entrada.close();
    }
    
}

File: Python/ExerciciosPython/Exercicio13.py
# Algoritmo que lê variáveis 'codigo' e 'horas_trab' de um operário, calcula o salário, sabendo que:
# Ele ganha R$10 por hora. Quando exceder 50hrs, calcula o excesso de pagamento e armazenando
# na variável 'excesso'. Caso contrário, ele zera a variável. A hora excedente vale R$20.
# No final do processamento imprime o salário total e excedente.

# Variáveis
excesso = 0
valor_hora = 10.00
valor_hora_excedente = 20.00

# Entrada
codigo = float(input("Informe o código: "))
horas_trab = int(input("Informe o número de horas trabalhadas: "))

# Processamento
if (horas_trab > 50):
    excesso = (horas_trab - 50) * valor_hora_excedente
    salario = (50 * valor_hora) + excesso
    print("Salário Total R${0:.2f}".format(salario))
    print("Salário Excedente R${0:.2f}".format(excesso))
else:
    salario = (horas_trab * valor_hora)
    print("Salário Total R${0:.2f}".format(salario))
    print("Salário Excedente R${0:.2f}".format(excesso))